src/test/java/org/prebid/server/bidder/telaria/TelariaBidderTest.java

```diff
@@ -319,14 +321,32 @@ public void makeBidsShouldReturnEmptyListIfBidResponseSeatBidIsNull() throws Jso
     }
 
     @Test
-    public void makeBidsShouldReturnErrorIfNoBidsFromSeatArePresent() throws JsonProcessingException {
+    public void makeBidsShouldReturnEmptyListIfNoBidsFromSeatArePresent() throws JsonProcessingException {
         // given
-        final HttpCall<BidRequest> httpCall = givenHttpCall(null,
+        final BidRequest bidRequest = BidRequest.builder().build();
+        final HttpCall<BidRequest> httpCall = givenHttpCall(bidRequest,
                 mapper.writeValueAsString(BidResponse.builder()
                         .seatbid(singletonList(SeatBid.builder().build())).build()));
 
         // when
-        final Result<List<BidderBid>> result = telariaBidder.makeBids(httpCall, null);
+        final Result<List<BidderBid>> result = telariaBidder.makeBids(httpCall, bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).isEmpty();
+    }
+
+    @Test
+    public void makeBidsShouldReturnEmptyListIfNoImpsFromBidRequestArePresent() throws JsonProcessingException {
+        // given
+        final BidRequest bidRequest = BidRequest.builder().build();
+        final HttpCall<BidRequest> httpCall = givenHttpCall(bidRequest,
+                mapper.writeValueAsString(BidResponse.builder()
+                        .seatbid(singletonList(SeatBid.builder()
+                                .bid(singletonList(Bid.builder().build())).build())).build()));
+
+        // when
+        final Result<List<BidderBid>> result = telariaBidder.makeBids(httpCall, bidRequest);
```

Comment:
This is unreal scenario, due to https://github.com/prebid/prebid-server-java/blob/c6a298f674bcc44164d6337798499e297336658b/src/main/java/org/prebid/server/validation/RequestValidator.java#L149